setup_wordpress.sh

```diff
@@ -1,3 +1,90 @@
 #!/bin/bash
+set -e
+echo '################## LEMP Stack Installation ####################'
+echo "------------------ Updating APT ------------------"
+sudo apt-get -y update && sudo apt-get -y upgrade
 
-echo 'This script should install and setup Wordpress'
+echo "------------------ Installing Nginx ------------------"
+sudo apt-get -y install nginx
+
+echo "------------------ Configuring Firewall ------------------"
+sudo ufw allow 'Nginx Full'
+
+echo "------------------ Installing MYSQL ------------------"
+sudo apt-get -y install mysql-server
+
+echo "------------------ Creating WordPress Database ------------------"
+mysql_password=QpWo#2LuQ
+debconf-set-selections <<< "mysql-server mysql-server/root_password password $mysql_password"
+debconf-set-selections <<< "mysql-server mysql-server/root_password_again password $mysql_password"
+
+sudo mysql -u root -p$mysql_password << EOF
+CREATE DATABASE IF NOT EXISTS wordpress DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+CREATE USER IF NOT EXISTS 'wordpressuser'@'localhost' IDENTIFIED BY 'password';
+GRANT ALL PRIVILEGES ON wordpress.* to wordpressuser@'localhost';
+FLUSH PRIVILEGES;
+EOF
+
+echo "------------------ Installing PHP Packages ------------------"
+sudo apt-get -y install php-fpm php-mysql php-curl php-gd php-intl php-mbstring php-soap php-xml php-xmlrpc php-zip
+sudo systemctl restart php7.4-fpm
+
+echo "------------------ Configuring SSL Certificate ------------------"
+cp /vagrant/nginx-selfsigned.crt /etc/ssl/certs/
+cp /vagrant/nginx-selfsigned.key /etc/ssl/private/
+cp /vagrant/dhparam.pem /etc/nginx/
+
+echo "------------------ Copying SSL Configurations to Snippet ------------------"
+cp /vagrant/self-signed.conf /etc/nginx/snippets/
+cp /vagrant/ssl-params.conf /etc/nginx/snippets/
+
+echo "------------------ Creating WordPress Nginx Directory ------------------"
+WP_DIR=/var/www/wordpress
+if [ ! -d "$WP_DIR" ]; then
+    sudo mkdir /var/www/wordpress
```

Comment:
in if statement you use `WP_DIR` variable, but in mkdir command you use full path. Is there any special reason to do so?

Reply:
I could use the defined variable.